eth-contracts/contracts/DelegateManager.sol

```diff
@@ -86,6 +86,12 @@ contract DelegateManager is InitializableV2 {
     // delegator -> (service provider -> delegatedStake)
     mapping (address => mapping(address => uint256)) private delegateInfo;
 
+    // Delegator stake total by address
+    // delegator -> (totalDelegated)
+    // Note - delegator properties are maintained in a mapping instead of struct
+    // in order to facilitate extensibility in the future.
+    mapping (address => uint256) private delegatorTotals;
```

Comment:
nit - can we call this something better like "delegatorTotalStakes" or "delegatorTotalStakeAmounts"
or w/e

Reply:
and then corresponding same change to function name

Reply:
delegatorTotalStake